package.json

```diff
@@ -90,7 +91,6 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-native": "0.72.5",
-    "react-navigation": "^4.4.4",
     "react-redux": "^8.1.3",
     "redux": "^4.2.1",
     "redux-saga": "^1.2.3",
```

Comment:
do we need still the redux-saga?

Reply:
yes, only because of `eslint-config-txo-typescript-react`, because it uses `eslint-plugin-redux-saga`

Reply:
you should be able to do cleanup